packages/perseus/src/components/graph.tsx

```diff
@@ -317,6 +318,33 @@ class Graph extends React.Component<Props> {
             setDrawingAreaAvailable: this.props.setDrawingAreaAvailable,
         });
 
+        // Maybe move this out later
+        if (this.props.lockedShapes) {
+            for (const lockedShape of this.props.lockedShapes) {
+                switch (lockedShape.type) {
+                    case "point":
+                        graphie.circle(
+                            // center
+                            lockedShape.coord,
+                            // radius
+                            0.2,
+                            // style
+                            lockedShape.style,
+                        );
+                        break;
+                    default:
+                        // Ensure that we've handled all locked shape types
+                        // at development time. Remove this after all the
+                        // locked shape types are implemented.
```

Comment:
I think we should keep the exhaustiveness check indefinitely. It will ensure that future maintainers update this code if they add a new locked shape type.